test/operator/test_operator.py

```diff
@@ -574,70 +309,3 @@ def test_operator_on_subspace():
 
     h4 = nk.operator.Heisenberg(hi, g, acting_on_subspace=0)
     assert h4.acting_on == h1.acting_on
-
-
-def test_openfermion_conversion():
-    # skip test if openfermion not installed
-    pytest.importorskip("openfermion")
-    from openfermion.ops import QubitOperator, FermionOperator
-
-    # first term is a constant
-    of_qubit_operator = (
-        QubitOperator("") + 0.5 * QubitOperator("X0 X3") + 0.3 * QubitOperator("Z0")
-    )
-
-    # no extra info given
-    ps = nk.operator.PauliStrings.from_openfermion(of_qubit_operator)
-    assert isinstance(ps, nk.operator.PauliStrings)
-    assert isinstance(ps.hilbert, nk.hilbert.Qubit)
-    assert ps.hilbert.size == 4
-
-    # number of qubits given
-    ps = nk.operator.PauliStrings.from_openfermion(of_qubit_operator, n_qubits=6)
-    assert isinstance(ps, nk.operator.PauliStrings)
-    # check default
-    assert isinstance(ps.hilbert, nk.hilbert.Qubit)
-    assert ps.hilbert.size == 6
-
-    # with hilbert
-    hilbert = nk.hilbert.Spin(1 / 2, 6)
-    ps = nk.operator.PauliStrings.from_openfermion(hilbert, of_qubit_operator)
-    assert ps.hilbert == hilbert
-    assert ps.hilbert.size == 6
-
-    # FermionOperator
```

Comment:
@wdphy16 you moved a little bit too many tests. From this line onwards they should go in `test_fermion`, not `test_pauli`

Reply:
thanks